lib/common/actions/index.js

```diff
@@ -49,14 +49,21 @@ export function secureFetch (url: string, method: string = 'get', payload?: any,
       .then(res => {
         // if raw response is requested
         if (raw) return res
-        let action, message
+        let action
         // check for errors
         const {status} = res
         if (status >= 500) {
           action = 'RELOAD'
-          message = `Network error!\n\n(${method} request on ${url})`
-          consoleError(message)
-          dispatch(setErrorMessage({message, action}))
+          res.json().then(json => {
+            const {detail, message} = json
+            const unknown = `Network error!\n\n(${method} request on ${url})`
+            consoleError(message || JSON.stringify(json) || unknown)
+            dispatch(setErrorMessage({
+              message: message || JSON.stringify(json) || unknown,
```

Comment:
Maybe the old message should still be shown for reference underneath the JSON data?

Reply:
As in below this for a combination of the two images in the issue screenshots? 
![image](https://user-images.githubusercontent.com/63798641/127356853-a71ecdfc-9495-487b-a0b3-3064bb329198.png)

Reply:
That is my thought, possibly with the original error in small font.

Reply:
Ok, I'll add that

Reply:
Added in https://github.com/ibi-group/datatools-ui/pull/694/commits/795d77430c6d2cfd9c69c536114cb6eac9f830d5

Reply:
However, @landonreed, do you have an opinion on including this? Output seems like it could start getting pretty verbose: 
![image](https://user-images.githubusercontent.com/63798641/127704977-3ecd383e-612f-4506-a418-de05666664e4.png)